### PR TITLE
Update system test fixtures due to electrode bridge upgrade to 1.5.27

### PR DESCRIPTION
**Pull request by jw2175**

### CODE REVIEW

**Comment by friederbluemle** [comment]
Somehow I'm still getting some changes even with this commit. Can you check please?
Is it something in my environment?

```
6 files changed, 16 insertions(+), 16 deletions(-)
```

https://github.com/friederbluemle/electrode-native/commit/b319a5401444bcbdea58f6c579ffd439fe2936ea

File: system-tests/fixtures/ios-container/container-metadata.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   ],
   "nativeDeps": [
     "react-native@0.63.2",
-    "react-native-electrode-bridge@1.5.26",
+    "react-native-electrode-bridge@1.5.27",
```
Comment (friederbluemle) [inline system-tests/fixtures/ios-container/container-metadata.json:10]: This also needs to be updated in

```
system-tests/fixtures/android-container/container-metadata.json
```

**Comment by friederbluemle** [comment]
Oh, I think you just need to regenerate api-impl-native as well (plus the one change in the android container).

**Comment by jw2175** [comment]
@friederbluemle can you check again?

**Review by friederbluemle — APPROVED** [review]
Looks good, thanks @jw2175 :+1:

**Pull request merged**